core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -85,29 +96,30 @@ public abstract class AbstractClient implements Client {
   private final long mRpcThreshold;
 
   /**
-   * Creates a new client base.
+   * Creates a new client base with default retry policy supplier.
    *
    * @param context information required to connect to Alluxio
    * @param address the address
    */
-  public AbstractClient(ClientContext context, InetSocketAddress address) {
+  public AbstractClient(ClientContext context, @Nullable InetSocketAddress address) {
```

Comment:
I don't think allowing nullable arguments in public constructors is a good idea. If this parameter can be null, provide an overloaded constructor without this parameter, and internally calling a private constructor that handles nullable arguments.